src/main.rs

```diff
@@ -91,14 +90,15 @@ fn print_stats(net: &run::Net, start_time: std::time::Instant) {
 // Load file and generate net
 fn load(file: &str) -> (run::Book, run::Net) {
   let file = fs::read_to_string(file).unwrap();
-  let book = ast::book_to_runtime(&ast::do_parse_book(&file));
+  let book = ast::book_to_runtime(&ast::do_parse_book(&file), run::call_native());
   let mut net = run::Net::new(1 << 28);
   net.boot(ast::name_to_val("main"));
   return (book, net);
 }
 
 pub fn compile_book_to_rust_crate(f_name: &str, book: &run::Book) -> Result<(), std::io::Error> {
```

Comment:
Will this functionality be kept even after we have jit compiling?

Reply:
yes i think, because jit is not fast as compiled code